gammapy/stats/variability.py

```diff
@@ -62,6 +64,62 @@ def compute_fvar(flux, flux_err, axis=0):
     return fvar, fvar_err
 
 
+def compute_fpp(flux, flux_err, axis=0):
+    r"""Calculate the point-to-point excess variance.
+
+    This method accesses the ``FLUX`` and ``FLUX_ERR`` columns
+    from the lightcurve data.
+
+    F_pp is a quantity strongly related to F_var, probing the variability
+    in a shorter timescale.
+
+    For white noise, F_pp and F_var give the same value.
+    However, for red noise, F_var will be larger
+    than F_pp, as the variations will be larger on longer timescales
+
+    It is important to note that the errors on the flux must be gaussian.
```

Comment:
@Astro-Kirsty , is it not here `Gaussian'?

Reply:
Oops, no disrespect to Gauss intended, I will capitalize it!